build.gradle

```diff
@@ -147,6 +172,31 @@ pmd {
     ruleSetFiles = files("config/pmd/ruleset.xml")
 }
 
+runAndPublishConsumerPactTests.dependsOn contract
+
+runAndPublishConsumerPactTests.finalizedBy pactPublish
+
+project.ext {
+  pacticipantVersion = getCheckedOutGitCommitHash()
+}
```

Comment:
I believe, custom property can be added in this manner:
`project.ext.participantVersion = getCheckedOutGitCommitHash()`
And if that works, I think it is nicer.

Reply:
Done